README.md

```diff
@@ -1,18 +1,25 @@
-# COVID Response App
+# COVID Response Toolkit
 
-Covid Response App is an open-source COVID-19 repository using [Hapi-FHIR](https://hapifhir.io/) as a [FHIR](https://www.hl7.org/fhir/) server and [Angular](https://angular.io/) for front-end application.
+Covid Response App is an open-source COVID-19 repository built with [Angular.js](https://angular.io/) for front-end application and [FHIR-R4](https://hl7.org/fhir/)-compliant backend API for interoperability.
```

Comment:
Angular.js is a term specifically used for angular 1, we're currently using Angular 9.